backend/src/case_file/case_file.graphql

```diff
@@ -143,10 +144,57 @@ type Query {
 type Mutation {
   createAssessment(createAssessmentInput: CreateAssessmentInput!): CaseFile!
   updateAssessment(updateAssessmentInput: UpdateAssessmentInput!): CaseFile!
+
   createPrevention(createPreventionInput: CreatePreventionInput!): CaseFile!
   updatePrevention(updatePreventionInput: UpdatePreventionInput!): CaseFile!
+
   createReview(reviewInput: ReviewInput!): CaseFile!
   updateReview(reviewInput: ReviewInput!): CaseFile!
+
   createNote(input: CreateSupplementalNoteInput!): CaseFile!
   updateNote(input: UpdateSupplementalNoteInput!): CaseFile!
+
+  createEquipment(createEquipmentInput: CreateEquipmentInput!): CaseFile!
+  updateEquipment(updateEquipmentInput: UpdateEquipmentInput!): CaseFile!
+}
+
+type EquipmentDetails {
+  equipmentGuid: String
+  actionEquipmentTypeCode: String
+  actionEquipmentTypeActiveIndicator: Boolean
+  address: String
+  xCoordinate: String
+  yCoordinate: String
+  actions: [Action]
+}
+
+input CreateEquipmentInput {
+  leadIdentifier: String
+  createUserId: String
+  agencyCode: String
+  caseCode: String
+  equipment: [EquipmentDetailsInput]
+}
+
+input UpdateEquipmentInput {
+  caseIdentifier: String!
+  equipment: EquipmentDetailsInput
+  updateUserId: String!
+}
+
+input EquipmentDetailsInput {
+  equipmentGuid: String
+  actionEquipmentTypeCode: String
```

Comment:
Same as above both of these two properties should be renamed `equipmentTypeCode` and `activeIndicator`